src/elm/Page/Community/New.elm

```diff
@@ -559,7 +559,12 @@ validateModel shared accountName model =
                         , hasObjectives = True
                         , hasKyc = False
                         , hasAutoInvite = model.hasAutoInvite
-                        , website = model.website
+                        , website =
+                            if String.startsWith "https://" model.website || String.startsWith "http://" model.website then
+                                model.website
+
+                            else
+                                "https://" ++ model.website
```

Comment:
i'm not sure about this, you see not all websites have https. good websites will automatically redirect to https:// when accessed on http

we do that!

Reply:
If we don't add the protocl (`https://`/`http://`) at the start, we get [the issue Karla talked about in Slack](https://cambiatus.slack.com/archives/CA83HJAAD/p1622157075032400), where the browser understands the url as an internal url, so it redirects to something like `ticolones.demo.cambiatus.io/www.ticolones.com` instead of redirecting to the external url `www.ticolones.com`. We could change it in the href instead, but it's kind of the same thing anyway, and we'd need to do it everywhere else we might use the community's website

Reply:
So the website doesn't even have the chance to redirect to `https://`, because the user isn't even redirected to it

Reply:
Oh, ok :sweat_smile: